README.md

```diff
@@ -81,11 +86,49 @@ For the full list of available restrictions see [`dscp-node`](https://github.com
 
 ### Create Process
 
+Takes multiple options which be parsed when initiating a new instance of `polkadot` API. Also, it takes a `dryRun` [true, false] which is basically a tests run that won't be executed against blockchain, default value is false
+
+```sh
+# commander CLI interface
+$ process-management help
+  ____                                                ____   _       ___ 
+ |  _ \   _ __    ___     ___    ___   ___   ___     / ___| | |     |_ _|
+ | |_) | | '__|  / _ \   / __|  / _ \ / __| / __|   | |     | |      | | 
+ |  __/  | |    | (_) | | (__  |  __/ \__ \ \__ \   | |___  | |___   | | 
+ |_|     |_|     \___/   \___|  \___| |___/ |___/    \____| |_____| |___|
+                                                                         
+Usage: process management [options] [command]
+
+a command line interface for managing chain processes
+
+Options:
+  -V, --version              output the version number
+  -h, --help                 display help for command
+
+Commands:
+  create [options] <string>  A command for persisting process flows onto the chain
+  disable
+  help [command]             display help for command
+
+# example of nothing to process result
+$ process-management create -p 20 -h paulius.local '[{}]'
+  ____                                                ____   _       ___ 
+ |  _ \   _ __    ___     ___    ___   ___   ___     / ___| | |     |_ _|
+ | |_) | | '__|  / _ \   / __|  / _ \ / __| / __|   | |     | |      | | 
+ |  __/  | |    | (_) | | (__  |  __/ \__ \ \__ \   | |___  | |___   | | 
+ |_|     |_|     \___/   \___|  \___| |___/ |___/    \____| |_____| |___|
+                                                                         
+parsed options:  {
+  options: { dryRun: false, host: 'paulius.local', port: '20', user: '//Alice' }
```

Comment:
`localhost`
port should be the dscp-node default rpc port e.g `9944`

Reply:
this is me setting up a port e.g. passing a custom one, maybe 20 is not a good example as it's one of the reserved ones